Recolor.php

```diff
@@ -27,6 +27,10 @@ public function image($imageName, $newFilename)
     {
         $imagePath = $this->imageFolder . $imageName;
 
+        if (!file_exists($imagePath)) {
+            throw new \Exception("Cannot find the file `$imagePath`.");
```

Comment:
I am a big fan of really verbose and descriptive error messages, so here we include the full file path, so the dev can see if there is a typo or similar in the filename.